Server/sprawl_calculator/wup_calculator/views.py

```diff
@@ -9,5 +12,13 @@ def wup_index_generator(request):
 
 #Endpoint2_View
 @api_view(['GET'])
-def index_calculation_status(request,processing_id):
-    raise NotImplementedError
\ No newline at end of file
+def index_calculation_status(request, processing_id):
+    try:
+        wup_calculation = WUPCalculation.objects.get(processing_id=processing_id)
+    except WUPCalculation.DoesNotExist:
+        return Response(status=status.HTTP_404_NOT_FOUND)
+    if wup_calculation.status == "Completed":
+        data = WUPCalculation.values()
+        return JsonResponse(data, status=status.HTTP_200_OK)
+    else:
+        return Response(status=status.HTTP_202_ACCEPTED)
```

Comment:
Lets return the appropriate object per the documentation e.g. if a valid processing_id is passed and the job status is `processing` then the [WUPCalculationRequestProcessing](https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L252) response should be returned, so depending on the status the appropriate response should be returned.